node/src/types/block.rs

```diff
@@ -1250,7 +1248,7 @@ pub(crate) mod json_compatibility {
                 hash: block.hash,
                 header: JsonBlockHeader::from(block.header),
                 body: block.body,
-                proofs: block.proofs.into_iter().map(JsonProof::from).collect(),
+                proofs: vec![],
```

Comment:
Why are you returning an empty vec?

Reply:
Well, the JSON block contains proofs but not the actual block, so I generate a JSON block with empty proofs, and then append the signatures to the JSON block separately